network/src/sync/blocks.rs

```diff
@@ -75,6 +76,8 @@ impl Node {
             )));
         }
 
+        // Set to `true` if the block was sent in a `Block` messages, `false` if it was sent in a
+        // `SyncBlock` message.
```

Comment:
that name isn't great; I think the `is_block_new` variable can have its name changed to `is_non_sync` in order to avoid any confusion